paddle/fluid/framework/parallel_executor.cc

```diff
@@ -218,8 +218,9 @@ void ParallelExecutor::BCastParamsToDevices(
         platform::NCCLGroupGuard guard;
         for (size_t i = 0; i < member_->places_.size(); ++i) {
           auto &nccl_ctx = member_->nccl_ctxs_->at(member_->places_[i]);
-          platform::dynload::ncclBcast(buffers[i], numel, data_type, 0,
-                                       nccl_ctx.comm_, nccl_ctx.stream());
+          PADDLE_ENFORCE(
```

Comment:
CUDNN_ENFORCE(...)

Reply:
Sorry, it should not be fixed in this PR!